src/main/kotlin/nl/info/zac/app/admin/ZaaktypeBpmnConfigurationRestService.kt

```diff
@@ -42,7 +42,6 @@ class ZaaktypeBpmnConfigurationRestService @Inject constructor(
 ) {
     @GET
     fun listZaaktypeBpmnConfigurations(): List<RestZaaktypeBpmnConfiguration> {
-        assertPolicy(policyService.readOverigeRechten().beheren)
```

Comment:
Shouldn't we at least check on the lezen policy? I.e. the raadpleger? Removing all policy checks seems a bit drastic or not?

Reply:
Still. We do not really have `lezen` without zaaktype and here we list all zaaktypes configurations

I think `startenZaak` would be appropriate @edgarvonk ?